velox/functions/lib/aggregates/tests/utils/AggregationTestBase.cpp

```diff
@@ -539,6 +539,26 @@ class ScopedChange {
   T* value_;
 };
 
+bool isTableScanSupported(const TypePtr& type) {
```

Comment:
nit: FYI, AggregationFuzzerBase has a similar function, ( I know these are both different packages ).  
Would also be nice to add in description that we have to add this support as checksum tests use tablescan.